opam HEAD, can't upgrade anything anymore.

```
> opam upgrade
The following actions will be performed:
 - recompile react.1.1.0*                          [upstream changes]
 - recompile gg.0.8.0*                             [upstream changes]
 - upgrade   optcomp from 1.5 to 1.6
 - recompile pkgopkg.repo                          [upstream changes]
 - upgrade   ocamlmod from 0.0.4 to 0.0.7
 - upgrade   ocamlnet from 3.7.3 to 3.7.4
 - recompile fut.unreleased*                       [uses react]
 - recompile lit.unreleased*                       [uses gg]
=== ↻ 5   ↑ 3 ===
Do you want to continue ? [Y/n] Y

🐫                   --- Synchronizing package archives ---
fut.unreleased Synchronizing with /Users/dbuenzli/sync/repos/fut
gg.0.8.0   Synchronizing with /Users/dbuenzli/sync/repos/gg
lit.unreleased Synchronizing with /Users/dbuenzli/sync/repos/lit
[ERROR] # opam-version    1.2.0
# os              darwin
opam: "wait" failed: Interrupted system call
```

--- a/src_ext/Makefile
+++ b/src_ext/Makefile
@@ -17,8 +17,8 @@ MD5_graph = 75dde65bfc3f9b07e795343d369aa84d
 URL_cudf = https://gforge.inria.fr/frs/download.php/file/33593/cudf-0.7.tar.gz
 MD5_cudf = 2047222fcf78278c6a24ac619fc39abb
 
-URL_dose = https://gforge.inria.fr/frs/download.php/file/33607/dose3-3.2-rc2.tar.gz
-MD5_dose = f2d229129f03ad0971fc9d68f4a90898
+URL_dose = https://gforge.inria.fr/frs/download.php/file/33677/dose3-3.2.2.tar.gz
+MD5_dose = 3ffed1987a040024076c08f4a7af9b21
 
 ARCHIVES = $(foreach lib,$(SRC_EXTS),$(notdir $(URL_$(lib))))
 lib_of = $(foreach lib,$(SRC_EXTS),$(if $(findstring $(1),$(URL_$(lib))),$(lib),,))

